config/routes.rb

```diff
@@ -2,6 +2,7 @@
   get "/pages/:page", to: "pages#show"
 
   resource :supplier_dashboard, controller: :supplier_dashboard, only: :show
+  resource :invite_schools, only: %i[show create]
```

Comment:
I think we want a noun as the resource and not a verb so probably school_invites